arctic-report-card/ARCRC.ttl

```diff
@@ -766,6 +936,152 @@ odo:ARCRC_00000068 rdf:type owl:Class ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Frostbite, 'Glacier and Permafrost Hazards'" ;
+                   rdfs:label "glacier and permafrost hazards"@en . 
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000070
+odo:ARCRC_00000070 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "A carbon flux refers to the amount of carbon exchanged between carbon stocks over a specified time. In simple terms, it is the movement of carbon between land, oceans, atmosphere, and living things." ;
+                   obo:IAO_0000119 "carbon balance as defined at https://www.canr.msu.edu/news/carbon-fluxes-and-carbon-stocks#" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Other Indicator, 'Arctic Carbon Cycling'" ;
+                   rdfs:label "carbon flux"@en ;
+                   skos:equivalentClass "http://purl.dataone.org/odo/ECSO_00001105" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000071
+odo:ARCRC_00000071 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "Data for each seal included collection date, species, sex, standard length, and blubber thickness." ;
+                   obo:IAO_0000119 "ice seal vital rates as defined in the Arctic Report Card 2024 Essay Ice Seals" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Other Indicator, 'Ice Seals'" ;
+                   rdfs:label "ice seal vital rates"@en . 
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000072
+odo:ARCRC_00000072 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "The atmospheric pressure at mean sea level, either directly measured or, most commonly, empirically determined from the observed station pressure." ;
+                   obo:IAO_0000119 "mean sea level pressure as defined at https://glossary.ametsoc.org/wiki/Sea_level_pressure" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Vital Sign, 'Surface Air Temperature'" ;
+                   rdfs:label "mean sea level pressure"@en ;
+                   skos:narrowMatch "http://purl.dataone.org/odo/ECSO_00001650" .
```

Comment:
I really like it when we can make these linkages to existing terms. Can you describe why this isn't an exact match?

Reply:
This was in reference to mean sea level pressure. You're right, it should be an exact match. I thought "mean sea level pressure" was more narrow than "sea level pressure", but I see that the definition for sea level pressure mentions it's the average pressure